Resources/public/webcomponents/ez-universal-discovery.html

```diff
@@ -0,0 +1,47 @@
+<link rel="import" href="../../../assets/ezplatform/polymer/polymer-element.html">
+<link rel="import" href="../../../webcomponents/ez-yui-app.html">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/finder.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/explorer.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/explorerlevel.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/search.css">
+<link rel="stylesheet" href="../../../bundles/ezplatformui/css/views/universaldiscovery/selected.css">
+<link rel="import" href="mixins/ez-sandbox-yui-component.html">
+
+<style>
+    ez-universal-discovery {
+        display: block;
+        background: rgba(0, 0, 0, 0.4);
+
+        /* taken from universaldiscovery.css .ez-universaldiscovery-container */
```

Comment:
you should remove that comment and clean up universaldiscovery.css for unncessary element (.ez-universaldiscovery-container is gone for instance)